Makefile

```diff
@@ -235,20 +233,18 @@ QEMU = $(shell which qemu-system-$(QEMU_ARCH))
 # Run Panix in QEMU
 .PHONY: run
 run: $(PRODUCTS_DIR)/$(RUNIMG)
-	$(QEMU)                           \
+	$(QEMU)                                      \
 	-drive file=$<,index=0,media=disk,format=raw \
 	$(QEMU_FLAGS)
 
 # Open the connection to qemu and load our kernel-object file with symbols
 .PHONY: run-debug
 run-debug: $(PRODUCTS_DIR)/$(RUNIMG)
 	# Start QEMU with debugger
-	($(QEMU)   \
+	$(QEMU)   \
 	-S -s      \
 	-drive file=$<,index=0,media=disk,format=raw \
-	$(QEMU_FLAGS) > /dev/null &)
-	sleep 1
-	wmctrl -xr qemu.Qemu-system-$(QEMU_ARCH) -b add,above
```

Comment:
RIP my favorite part of this recipe 😭

Reply:
Yeah, sorry. If we want, we could maybe add it back with detection code to see if `wmctrl` exits?